src/hooks/ENS/contracts/PublicResolverAbi.json

```diff
@@ -1,5 +1,4 @@
 {
-  "address": "0x19c2d5D0f035563344dBB7bE5fD09c8dad62b001",
```

Comment:
we no longer need to store an address for the Resolver contract. instead we use a contract call to get the address for a specific ENS node's resolver